### DISCUSSION TITLE
How do I show priority load?

### DISCUSSION THREAD

**Discussion by perjmolsen** [slipx06/sunsynk-power-flow-card#209] [Q&A] [answered]
Hello

I can see in the documentation that it is possible to show the inverters priority load, but I am unable to get it working :(

I have set the priority_load_243: to my inverters device mode, but no icon is beeing showed at the sunsynk-power-flow card

In the card I have set:

priority_load_243: sensor.growatt_sph_device_mode
This sensor displays: Battery, Load or Grid
![sunsynk](https://github.com/slipx06/sunsynk-power-flow-card/assets/126662740/53fe0e78-699d-442d-8bde-ab4473a10d56)

**Comment by slipx06** [comment]
Hi. What data does your sensor `sensor.growatt_sph_device_mode` provide. The card expects this to be either `on` or `off`.

**Comment by perjmolsen** [comment]
Oh thats might be why it does not work then.
My sensor provides the following information
Load First
Battery First
Grid First

Is there another place in the card where I can show the information of my sensor?

**Comment by slipx06** [comment, marked as answer]
. No there is no other place. You can create a template sensor that maps "Load First" to on and the other two options to off

**Comment by perjmolsen** [comment]
Thanks :) I have added a template sensor - and It works perfect!
Thanks for your great card
![Priority load](https://github.com/slipx06/sunsynk-power-flow-card/assets/126662740/88b7f02e-167f-49a4-a95b-fa221d2855a6)
